README.md

```diff
@@ -1 +1,35 @@
-# ethics-in-data-viz
\ No newline at end of file
+# Ethics in Data Visualization workshop
+
+## Introduction
+This hour-long workshop (undergrad and grad level) provides an introduction to ethics in data visualization. Please feel free to use this workshop, repurpose and remix this content, add suggestions to this project, etc.!
+
+This work is part of the IMLS-funded [Visualizing the Future](visualizingthefuture.github.io/) project.
```

Comment:
The official statement should be:

This project was made possible in part by the [Institute of Museum and Library Services](https://www.imls.gov/), RE-73-18-0059-18.